test/orm/test_session.py

```diff
@@ -609,6 +609,24 @@ def check(ctx: ORMExecuteState) -> None:
 
         is_true(called)
 
+    def test_get_one(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4872